src/index.html

```diff
@@ -17,16 +17,29 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
-
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="form-search-bar form-search-bar--big"
+      data-qa="big"
+    >
+      <input
+        class="input-bar input-bar--big"
+        id="input-icon-big"
+        type="text"
+        data-qa="keypress"
```

Comment:
The `data-qa` attribute is used here, but it's not a standard HTML attribute. Ensure that this is intended for a specific purpose, such as for testing or tracking, and that it is supported by your framework or testing tools.